janus_client/Cargo.toml

```
@@ -7,7 +7,7 @@ rust-version = "1.58"
 
 [dependencies]
 http = "0.2.8"
-janus = { path = "../janus" }
+janus = "0.1.0"
```

The error message I got says the following:
```
$ cargo publish -p janus_client --dry-run
    Updating crates.io index
error: all dependencies must have a version specified when publishing.
dependency `janus` does not specify a version
Note: The published dependency will use the version from crates.io,
the `path` specification will be removed from the dependency declaration.
```

We can thus use something like the following:

```suggestion
janus = { version = "0.1.0", path = "../janus" }
```

During normal development, Cargo will use the path dependency (or error if the version doesn't match, I think) and during publishing, it will rewrite the Cargo.toml file to refer to `crates.io` alone. (It looks like it also does a check build with `crates.io` dependencies, as I get errors from it trying to use the unrelated existing janus crate)